synapse/models/inet.py

```diff
@@ -1530,6 +1533,14 @@ def getModelDefs(self):
                         'interfaces': ('inet:service:object',),
                         'doc': 'An account within a service platform. Accounts may be instance specific.'}),
 
+                    ('inet:service:relationship:type:taxonomy', ('taxonomy', {}), {
+                        'interfaces': ('meta:taxonomy',),
+                        'doc': 'A service object relationship type taxonomy.'}),
+
+                    ('inet:service:relationship', ('guid', {}), {
```

Comment:
Since the relationship is account specific, should this be 'inet:service:account:relationship'? The name is long, but to me it makes it clear that the relationship is not about the service.